### ISSUE TITLE
R.U.B.I.E. On Rails

### ISSUE THREAD

**Issue by week3** [siggame/MegaMinerAI-Game-Ideas#50]
# R.U.B.I.E. On Rails
Rossum's universal biometric intelligent engines on rails.
## TLDR
Turn based train game. Get money by transporting civilians and spend it on new tracks or destroying your competition.
## Story
Super intelligent AIs were created to run railways and they went rouge because syfy. However, they didn't go kill all humans rouge they just stopped listening to humans and now only care about profits. Yay? Anyways, after extreme lobbying these AIs were allowed to remain as long as they kept making profits under the company name. You are one of these AIs and you are in an area with only one competitor... You know what to do.
## Win Conditions
Destroy opponents train yard.
**If the turn limit is reached then victory will be decided with these conditions:**

1.   The player with the most money.
2.   The player with the most capital (cost of trains).
3.   The player with the highest reputation.
4.   Coin toss.
### The field
Each player starts on opposite sides of a 21 by 20 grid field with a train yard. The field is semi randomly filled with train stations and mountains.
### Train yard
Players home base. If this is destroyed they die. Starts with 3 rails one in front of it, to its left, and to its right. These starting rails cannot be destroyed. Players can spawn new trains on any track touching the yard using their money.
### Trains

Train | Description
-- | --
Transport | Can move civilians
Builder | Can make tracks
Gunner | Can attack units/buildings at range for medium damage
Battering Ram | Can attack units/building its touching for high damage

### Train stations
Not owned by either player. Civilians can be picked up and dropped off at these stations. Players are awarded money based on the distance between stations not the length of the ride.
### Rails
Rails can be used by both players regardless of who made them. However rail switches connecting tracks in different directions can only be controlled by that player on their turn. Aka you cant set another players switch to go forward instead of left unless you destroy and rebuild them. These switches cost more to make then regular rails. Further trains cannot move into the same rail tile.
### Combat
Trains that can attack can destroy: other trains, train yards, and tracks/switches. 
Combat trains can be set to attack specific trains that go by.
### Popularity
This is how much civilians trust your company. It starts at 100% and can be anywhere from 0%-200%. How much money you get per load is modified by this. 

You can lose popularity if a train you were transporting units in gets destroyed. You lose even more if you are the one to destroy a train carrying civilians.

You gain popularity for successfully transporting civilians with no one dying.
### Unmarked Trains
Unmarked variants of combat trains can be made that don't cause you to decrease in popularity for loaded transports destroyed. However they will be detected and eliminated by government forces in 5 turns.
### Turns
This game is turn based so one player starts doing everything they can or want to do in their turn then passes the turn to their opponent.

**Comment by JeffreyStrahm** [comment]
So, if I am understanding this correctly, you must build your own rails to go grab civilians from stations and transport them to other stations to make money. You can also only move on these rails and only builders can build the rails. You can build switches to make them one way or just controllable. I assume they can be destroyed. A list of the things that are buildable would be useful. Overall my only concern with the game is we may end up with no player interaction, as they must build rails to get to each other to fight. Can you attack and destroy rails?
